scripts/exregional_run_post.sh

```diff
@@ -175,6 +175,14 @@ fi
 cp_vrfy ${post_config_fp} ./postxconfig-NT.txt
 cp_vrfy ${EMC_POST_DIR}/parm/params_grib2_tbl_new ./params_grib2_tbl_new
 cp_vrfy ${EXECDIR}/ncep_post .
+cp_vrfy ${FFG_DIR}/latest.FFG .
+if [ ${NET} = "RRFS_CONUS" ]; then
+  grid_specs_rrfs="lambert:-97.5:38.500000 237.826400:1746:3000 21.88589:1014:3000"
```

Comment:
Sorry to nitpick here.  These are rounded off and instead should exactly match the RRFS_CONUS specs.  Otherwise people are going to wonder why they don't (even though this works out to centimeters)
Here are the exact specs from a "wgrib2 -V" on a BGSFC file: lambert:-97.5:38.500000 237.826355:1746:3000 21.885885:1014:3000

Reply:
No this is great, thanks for keeping it going!   Good catch!  Let me fix that.